presto-test-jdbc-compatibility/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.prestosql</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>337-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-test-jdbc-compatibility</artifactId>
+    <description>Presto - Tests whether older Presto JDBC clients are compatible with current Presto server</description>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <dep.presto-jdbc-under-test>335-SNAPSHOT</dep.presto-jdbc-under-test>
+    </properties>
+
+    <dependencies>
```

Comment:
please put dependencies in order, presto, airlift and rest. Order them then alphabetically when reasonable.